guide/the-user-interface.md

```diff
@@ -390,7 +388,7 @@ The `License Manager...` item opens the `Webots License Manager` window that
 allows you to see which licenses are in use and optionally to transfer some
 license to your local computer for off-line use. Please note that the transfer
 of license may be limited by your local license administrator: It may not be
-possible to transfer some license to your local computer or only for a limited
+possible to transfer some licenses to your local computer or only for a limited
```

Comment:
then "some" should be changed to "a", unless it means has the meaning of "in some special cases ..."